Sources/MongoSwift/BSON/BSONValue.swift

```diff
@@ -572,6 +587,13 @@ public struct Decimal128: BSONNumber, Equatable, Codable, CustomStringConvertibl
      }
 }
 
+// An extension of `Decimal128` to add capability to be hashed
+extension Decimal128: Hashable {
+    public func hash(into hasher: inout Hasher) {
+        hasher.combine(description)
```

Comment:
nit: let's say `self.description` - although the compiler doesn't require it, we try to always explicitly use `self` so it's clearer where the value is coming from.

Reply:
note that there is a swiftlint rule for this but it is _very_ bad at actually catching violations of the rule...